### PR TITLE
Update MobileFuse SDK to v1.6.0

### PR DESCRIPTION
**Pull request by tomaszbmf**
+ Update MobileFuse SDK to v1.6.0
+ Init MobileFuse SDK in the Adapter setUp call

### CODE REVIEW

File: MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt [hunk 1]
```diff
@@ -115,8 +115,24 @@ class MobileFuseAdapter : PartnerAdapter {
...
+                }
+            }
+
+            MobileFuse.init(object : SdkInitListener {
+                override fun onInitSuccess() {
+                    resumeOnce(Result.success(PartnerLogController.log(SETUP_SUCCEEDED)))
+                }
+
+                override fun onInitError() {
+                    resumeOnce(Result.success(PartnerLogController.log(SETUP_FAILED)))
```
Comment (cb-jpadilla) [inline MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt:131]: Let's pass a `Result.failure` here.
```suggestion
PartnerLogController.log(SETUP_FAILED)
resumeOnce(
    Result.failure(
        ChartboostMediationAdException(ChartboostMediationError.CM_INITIALIZATION_FAILURE_UNKNOWN)
    )
)
```

**Review by chauduyphanvu — CHANGES_REQUESTED** [review]
Besides @cb-jpadilla's point, please also update the CHANGELOG and README.

File: MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt (hunk 1 above)
Comment (tomaszbmf) [reply MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt]: Thanks, I added this change

File: MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt [hunk 2]
```diff
@@ -115,8 +115,29 @@ class MobileFuseAdapter : PartnerAdapter {
...
+
+                override fun onInitError() {
+                    PartnerLogController.log(SETUP_FAILED)
+                    resumeOnce(
+                        Result.failure(
+                            ChartboostMediationAdException(ChartboostMediationError.CM_INITIALIZATION_FAILURE_UNKNOWN)
+                        )
+                    )
+                }
+
```
Comment (chauduyphanvu) [inline MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt:138]: Remove extra line

**Review by chauduyphanvu — APPROVED** [review]
lgtm with a minor change. we'll run the code changes through certification. once everything looks good, they'll be released. thanks!

File: MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt (hunk 2 above)
Comment (tomaszbmf) [reply MobileFuseAdapter/src/main/java/com/chartboost/mediation/mobilefuseadapter/MobileFuseAdapter.kt]: ok, done

**Review by cb-jpadilla — APPROVED** [review]
Looks good. Thank you for the changes. 👍🏼

**Pull request merged**